### PR TITLE
Honor the Start of Week for Admin Datepickers

### PR DESCRIPTION
**Pull request by mitogh**
Ticket: https://central.tri.be/issues/76320

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | There must be a single space before the last parenthesis | [Admin.php:28](https://github.com/moderntribe/the-events-calendar/blob/8a22e05d02c06bd88796fbd5626718e0bffa8c37/src/Tribe/Asset/Admin.php#L28)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Asset/Admin.php [hunk 1]
```diff
@@ -22,7 +22,12 @@ public function handle() {
 
 		$path = Tribe__Events__Template_Factory::getMinFile( tribe_events_resource_url( 'events-admin.js' ), true );
 
-		wp_enqueue_script( $this->prefix . '-admin', $path, $deps, $this->filter_js_version(), true );
+
+		$handle = $this->prefix . '-admin';
+		wp_enqueue_script( $handle, $path, $deps, $this->filter_js_version(), true );
```
Comment (bordoni) [inline src/Tribe/Asset/Admin.php]: Can we branch this out of the code I did for #1892

File: src/resources/js/events-admin.js [hunk 2]
```diff
@@ -428,6 +428,10 @@ jQuery( document ).ready( function( $ ) {
 		}
 
 		var startofweek = 0;
+		if ( 'firstDay' in tribe_datepicker_opts ) {
+			var firstDay = parseInt( tribe_datepicker_opts.firstDay );
```
Comment (bordoni) [inline src/resources/js/events-admin.js]: pass the second param into `parseInt(  , 10 )`

**Comment by mitogh** [comment]
# Dependency

#1892

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | There must be no space between the &quot;array&quot; keyword and the opening parenthesis | [Assets.php:62](https://github.com/moderntribe/the-events-calendar/blob/59fc2adc99783d41145a65d8a8ebf49d8c8e8803/src/Tribe/Assets.php#L62)
:x: | There must be no space between the &quot;array&quot; keyword and the opening parenthesis | [Assets.php:66](https://github.com/moderntribe/the-events-calendar/blob/59fc2adc99783d41145a65d8a8ebf49d8c8e8803/src/Tribe/Assets.php#L66)
:x: | Each line in an array declaration must end in a comma | [Assets.php:68](https://github.com/moderntribe/the-events-calendar/blob/59fc2adc99783d41145a65d8a8ebf49d8c8e8803/src/Tribe/Assets.php#L68)
:x: | Comma required after last value in array declaration | [Assets.php:71](https://github.com/moderntribe/the-events-calendar/blob/59fc2adc99783d41145a65d8a8ebf49d8c8e8803/src/Tribe/Assets.php#L71)

> via jenkins/codesniffer-phpcs

**Comment by bordoni** [comment]
Moving this to _"Hold"_ until we have the dependency merged.

**Comment by tr1b0t** [comment]
Closing this Pull Request because it is pull requested against the wrong branch. The Central ticket is for *M18.06*, but this was pull requested against *release/M18.07*.

**Comment by tr1b0t** [comment]
Closing this Pull Request because it is pull requested against the wrong branch. The Central ticket is for *M18.08*, but this was pull requested against *release/M18.07*.

**Comment by masteradhoc** [comment]
@mitogh this shouldnt be closed right?

**Comment by mitogh** [comment]
That's right is closed by the bot automatically as should point to a new branch so will update accordingly @masteradhoc .

Thanks

**Pull request closed without merging**